dask/bag/avro.py

```diff
@@ -1,6 +1,10 @@
 import io
 import uuid
 
+from fsspec.core import open_files, get_fs_token_paths, OpenFile
+from fsspec.utils import tokenize, read_block
```

Comment:
Let's avoid a naming collision with dask.base.tokenize
```suggestion
from fsspec.utils import tokenize as fsspec_tokenize, read_block
```